wporg-gp-translation-events.php

```diff
@@ -12,14 +12,19 @@
  * @package Translation Events
  */
 
-require_once __DIR__ . '/includes/class-wporg-gp-translation-events-route.php';
+if ( class_exists( GP_Route::class ) ) {
```

Comment:
This is here because:

When wp-env boots, it will try to activate both GlotPress, and this plugin. However, GlotPress does not activate because the permalink structure setting is not correct. So the `GP_Route` class (and I assume all other GlotPress code?) is not available.

Since the class is not found, wp-env fails to start with an error. That's why I added this check here, and also below, in the `register_routes()` function.